Tasks/Sepseleva/Task4/Task4.py

```diff
@@ -0,0 +1,63 @@
+#getting data from the user + validation
+while True:
+    chunk = input('Enter the number of symbols in a line that is more than 35: ')
+
+    if not (chunk.isdigit()):
+        print('Your input should contain only numbers. Try again')
+        continue
+
+    chunk = int(chunk) 
+
+    if chunk < 35:
+        print('Your value should be more than 35. Try again')
+        continue
+
+    break  
+
+position = 0  #the position of the pointer in the file
+
+#counting the number of symbols in a string
+def count_length(l):
+    length =len(' '.join(l)) 
+    return length
+
+#reading the contents 
+with open('text.txt', 'r', encoding='utf-8') as text:
+    with open('new_text.txt', 'a', encoding='utf-8') as copy:
+        while True:
+            part = text.read(chunk+1)
+            if part:
+                if len(part) < chunk:
+                    line = part.split()
+                elif part[-1] != ' ':
+                    line = part.split()
+                    del line[-1]
+                else:
+                    part = part[:-1]
+                    line = part.split() 
```

Comment:
this block could be slightly reworked, you do the same action **line = part.split()** in every case, it could be placed outside of this **if**